### PR TITLE
add dropDownnumber of traveller choice

### PR DESCRIPTION
**Pull request by pjroggy**

### CODE REVIEW

**Review by lxndrMr — APPROVED** [review]

File: client/src/components/DropdownTraveller.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+function DropdownTraveller() {
+  const [selectedOption, setSelectedOption] = useState("");
+
+  const handelSelectChange = (event) => {
+    setSelectedOption(event.target.value);
+  };
+
+  return (
+    <section className="dropDownTravellerSection">
+      <label className="dropDownTravellerLabel" htmlFor="select-number-traveller">
```
Comment (Yoann-Fortin) [inline client/src/components/DropdownTraveller.jsx]: camelCase please here

File: client/src/components/DropdownTraveller.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        Nombre de passagers :
+      </label>
+      <select
+        name="travellers"
+        id="select-number-traveller"
+        value={selectedOption}
+        onChange={handelSelectChange}
+        className="dropDownTravellerSelect"
+      >
+        <option value={1} selected>
```
Comment (Yoann-Fortin) [inline client/src/components/DropdownTraveller.jsx]: to refactor please

File: client/src/styles/DropdownTraveller.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.dropDownTravellerSection {
+  margin: 50px;
+  width: 15rem;
+  padding: 0.8rem 0.2rem 0.8rem 0.5rem;
+  border-radius: 2rem;
+  background-color: #fff8f0;
```
Comment (Yoann-Fortin) [inline client/src/styles/DropdownTraveller.scss]: variable css unused here, why ?

File: client/src/styles/DropdownTraveller.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+}
+.dropDownTravellerLabel {
+  background-color: #fff8f0;
```
Comment (Yoann-Fortin) [inline client/src/styles/DropdownTraveller.scss]: same

File: client/src/styles/DropdownTraveller.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  border: none;
+}
+.dropDownTravellerSelect {
+  background-color: #fff8f0;
```
Comment (Yoann-Fortin) [inline client/src/styles/DropdownTraveller.scss]: same

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by lxndrMr — APPROVED** [review]

File: client/src/components/DropdownTraveller.jsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      </label>
+      <select
+        name="travellers"
+        id="selectNumberTraveller"
+        value={selectedOption}
+        onChange={handelSelectChange}
+        className="dropDownTravellerSelect"
+      >
+        {[...Array(10)].map((_, index) => (
+          <option key={Array.id} value={index + 1}>
```
Comment (Yoann-Fortin) [inline client/src/components/DropdownTraveller.jsx]: What is the value of this id?

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/components/DropdownTraveller.jsx (hunk 6 above)
Comment (pjroggy) [reply client/src/components/DropdownTraveller.jsx]: finaly i've created an array of 10 options and give the id of the array in map key

File: client/src/components/DropdownTraveller.jsx [hunk 7]
```diff
@@ -8,32 +8,27 @@ function DropdownTraveller() {
...
-        <option value={3}>3</option>
-        <option value={4}>4</option>
-        <option value={5}>5</option>
-        <option value={6}>6</option>
-        <option value={7}>7</option>
-        <option value={8}>8</option>
-        <option value={9}>9</option>
-        <option value={10}>10</option>
+        {optionArray.map((_, index) => (
+          <option key={optionArray.id} value={index + 1}>
```
Comment (Yoann-Fortin) [inline client/src/components/DropdownTraveller.jsx]: What is the value of this id ?

File: client/src/styles/DropdownTraveller.scss [hunk 8]
```diff
@@ -1,16 +1,18 @@
-.dropDownTravellerSection {
-  margin: 50px;
-  width: 15rem;
-  padding: 0.8rem 0.2rem 0.8rem 0.5rem;
-  border-radius: 2rem;
-  background-color: #fff8f0;
-}
+$whiteColor: #fff8f0;
```
Comment (Yoann-Fortin) [inline client/src/styles/DropdownTraveller.scss]: Not the good file for this

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/components/DropdownTraveller.jsx [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+      </label>
+      <select
+        name="travellers"
+        id="selectNumberTraveller"
+        value={selectedOption}
+        onChange={handelSelectChange}
+        className="dropDownTravellerSelect"
+      >
+        {optionArray.map((_, index) => (
+          <option key={optionArray.id} value={index + 1}>
```
Comment (Yoann-Fortin) [inline client/src/components/DropdownTraveller.jsx]: Why use index here and what is this value of id ?

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**